apps/qubit/modules/contactinformation/templates/_edit.php

```diff
@@ -19,7 +19,6 @@
         </th><th style="width: 15%">
           <?php echo __('Primary') ?>
         </th><th style="text-align: center; width: 10%">
-          <?php echo image_tag('delete', array('align' => 'top', 'class' => 'deleteIcon')) ?>
```

Comment:
They are being removed with JS, so I just remove the images, I'll remove the following line in the JS too:

https://github.com/artefactual/atom/blob/qa/2.3.x/js/multiDelete.js#L46